members/SWAU22467_226588.yaml

```diff
@@ -0,0 +1,72 @@
+bioguide: SWAU22467_226588
+contact_form:
+  driver: phantom
+  steps:
+    - visit: "https://app.leg.wa.gov/pbc/memberEmail/2/0"
+    - fill_in:
+        - name: street
+          selector: "#Address_Street"
+          value: $ADDRESS_STREET
+          required: true
+        - name: city
+          selector: "#Address_City"
+          value: $ADDRESS_CITY
+          required: true
+        - name: zipcode
+          selector: "#Address_Zip"
+          value: $ADDRESS_ZIP5
+          required: true
+    - click_on:
+        - value: Validate address
+          selector: "#ValidateAddressButton"
+    - fill_in:
+        - name: First Name
+          selector: "#FirstName"
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: "#LastName"
+          value: $NAME_LAST
+          required: true
+        - name: email
+          selector: "#Email"
+          value: $EMAIL
+          required: true
+        - name: confirmemailaddress
+          selector: "#ConfirmEmail"
+          value: $EMAIL
+          required: true
+        - name: phone
+          selector: "#PhoneNumber"
+          value: $PHONE
+          required: false
+        - name: subject
+          selector: "#CommentSubject"
+          value: $SUBJECT
+          required: true
+          options: 
+            max_length: 100
+        - name: message
+          selector: "#Comment"
+          value: $MESSAGE
+          required: true
+          options: 
+            max_length: 1000
```

Comment:
Didn't we see an issue where we need this to be a bit lower than the true max? Can you test with a message > 1000 chars and ensure this works?

Reply:
lowered this to 900 chars